app/model/lib/submission_process.py

```diff
@@ -400,14 +409,13 @@ def _save_measurements(db_session, study, submission):
 
 
 def _create_average_measurements(db_session, study, experiment):
-    bioreplicate_ids = [b.id for b in experiment.bioreplicates]
+    bioreplicate_ids = [b.id for b in experiment.bioreplicates if not b.calculationType]
```

Comment:
This shouldn't be a problem in practice, since bioreplicates get reset and only then averaged, but it doesn't hurt to throw in an extra condition -- we only average non-calculated bioreplicates. If at some point the order of operation changes, we want to make sure this logic accurately reflects the intent.